### PR TITLE
Prep for notifying speakers: feedback form and (radical) visting page

### PR DESCRIPTION
**Pull request by dcwalk**
This will address https://github.com/ournetworks/2019/issues/49!

### CODE REVIEW

File: sessions-form.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (garrying) [inline sessions-form.html:1]: @dcwalk should the filename be `session-confirmation`? Looking at the URL in https://github.com/ournetworks/2019/pull/66/files#diff-05c96faad7724d9a3e068b6976a623ddR14

File: js/main.js [hunk 2]
```diff
@@ -45,6 +45,10 @@ if (window.location.search == '?donation=confirmed') {
   formSubmitCallback('Thanks for your support!');
 }
 
+if (window.location.search == '?update=confirmed') {
+  formSubmitCallback('Successfully confirmed your session!');
```
Comment (garrying) [inline js/main.js:49]: Added a brief announcement banner message

File: visiting.md [hunk 3]
```diff
@@ -1,20 +1,60 @@
...
+## Staying in Toronto
 
-### What is the best way to get around Toronto?
+The city is an expensive place to live and visit. For those looking at alternative acccomodations including platforms like AirBnB, we wanted to provide context about the housing market. Like many cities around the world, Toronto is experiencing an [affordability crisis](https://www.blogto.com/city/2019/03/report-housing-affordability-crisis-toronto/) in part driven by [condos being treated as investments](https://www.theguardian.com/world/2019/jul/07/toronto-housing-owner-occupied-canada-affordability) and ["ghost hotels"](https://www.thestar.com/opinion/star-columnists/2017/12/01/haunted-by-the-rise-of-ghost-hotels.html). This has recently led to [renovictions](https://nowtoronto.com/news/renovicted-toronto-rental-housing/) and more recently [tenant organizing and rent strikes](http://parkdaleorganize.ca/).
```
Comment (garrying) [inline visiting.md]:
```suggestion
The city is an expensive place to live and visit. For those looking at alternative accommodations including platforms like Airbnb, we wanted to provide context about the housing market. Like many cities around the world, Toronto is experiencing an [affordability crisis](https://www.blogto.com/city/2019/03/report-housing-affordability-crisis-toronto/) in part driven by [condos being treated as investments](https://www.theguardian.com/world/2019/jul/07/toronto-housing-owner-occupied-canada-affordability) and ["ghost hotels"](https://www.thestar.com/opinion/star-columnists/2017/12/01/haunted-by-the-rise-of-ghost-hotels.html). This has recently led to [renovictions](https://nowtoronto.com/news/renovicted-toronto-rental-housing/) and more recently [tenant organizing and rent strikes](http://parkdaleorganize.ca/).
```

File: visiting.md [hunk 4]
```diff
@@ -1,20 +1,60 @@
...
-We are committed to reducing barriers to access throughout the event. All spaces—for Kickoff Yami-ichi, Conference, and Sprints—have elevator access and accessible washrooms.
+Billeting is a way of supporting participants to attend within their budget and at the same time, welcome them into our community. 
+
+We anticipate securing a limited number of spots with people based in Toronto. If you indicated you were interested in hosting or staying in a community billet check out the [community billets list](https://ethercalc.org/s0r2i3dsfznl) and add your info or email billets@ournetworks.ca and we’ll help coordinate!
+
+### Small-businesses and BIPOC-owned hotels
+
+- [Baldwin Village Inn](https://www.urbannorthinns.com/baldwin-village-inn/) and [Urban North Inns](https://www.urbannorthinns.com/), 9 Baldwin Street
+
+We searched and could not confirm a directory of BIPOC-owned accomodations, if you know of any, please contact us.
```
Comment (garrying) [inline visiting.md]:
```suggestion
We searched and could not confirm a directory of BIPOC-owned accommodations, if you know of any, please contact us. 
```

File: visiting.md [hunk 5]
```diff
@@ -1,20 +1,60 @@
...
+- [One King West Hotel & Residence](https://www.onekingwest.com/)
+- [The Omni King Edward Hotel](https://www.guestreservations.com/the-omni-king-edward-hotel/booking)
+- [Marriott St. Regis](https://www.marriott.com/hotels/travel/yyzxr-the-st-regis-toronto/)
+
+### Other Short-term rentals
+
+Additional care to take if engaging in a short-term rental in Toronto:
+
+1. Avoid hosts with multiple listings.
+1. Avoid AirBnB rentals in the following neighbourhoods:
```
Comment (garrying) [inline visiting.md]:
```suggestion
1. Avoid Airbnb rentals in the following neighbourhoods:
```

File: visiting.md [hunk 6]
```diff
@@ -1,20 +1,60 @@
...
+  - Parkdale
+1. Avoid these hosts identified by a local nonprofit as AirBnB "ghost hotel" landlords.
```
Comment (garrying) [inline visiting.md]:
```suggestion
1. Avoid these hosts identified by a local nonprofit as Airbnb "ghost hotel" landlords.
```

File: visiting.md [hunk 7]
```diff
@@ -1,20 +1,60 @@
...
+To support folks attending the conference, we have a [community billets](#community-billets) program and provide information on ways to mitigate the impact of short-term rentals. Inspired by [4S 2019 in New Orleans](https://www.4s2019.org/accommodation/), notes on how to make your own guide are shared at the bottom of this page.
 
-### Are the venues accessible?
+### Community billets
 
-We are committed to reducing barriers to access throughout the event. All spaces—for Kickoff Yami-ichi, Conference, and Sprints—have elevator access and accessible washrooms.
+Billeting is a way of supporting participants to attend within their budget and at the same time, welcome them into our community. 
+
+We anticipate securing a limited number of spots with people based in Toronto. If you indicated you were interested in hosting or staying in a community billet check out the [community billets list](https://ethercalc.org/s0r2i3dsfznl) and add your info or email billets@ournetworks.ca and we’ll help coordinate!
```
Comment (garrying) [inline visiting.md]:
```suggestion
We anticipate securing a limited number of spots with people based in Toronto. If you indicated you were interested in hosting or staying in a community billet check out the [community billets list](https://ethercalc.org/s0r2i3dsfznl) and add your info or email [billets@ournetworks.ca](mailto:billets@ournetworks.ca) and we’ll help coordinate!
```

File: visiting.md [hunk 8]
```diff
@@ -1,20 +1,60 @@
...
+
+### Small-businesses and BIPOC-owned hotels
+
+- [Baldwin Village Inn](https://www.urbannorthinns.com/baldwin-village-inn/) and [Urban North Inns](https://www.urbannorthinns.com/), 9 Baldwin Street
+
+We searched and could not confirm a directory of BIPOC-owned accomodations, if you know of any, please contact us. 
+
+### Unionized hotels
+
+We used [Fair Hotels](https://www.fairhotel.org/) and called []
```
Comment (garrying) [inline visiting.md]:
```suggestion
We used [Fair Hotels](https://www.fairhotel.org/) and Called [UNITE HERE 75](https://www.uniteherelocal75.org/) for leads and updates. As of May 2019, these hotels are still unionized with them:
```

File: visiting.md [hunk 9]
```diff
@@ -1,20 +1,60 @@
...
+Called [UNITE HERE 75](https://www.uniteherelocal75.org/) for leads and updates, As of May 2019, these hotels are still unionized with them:
```
Comment (garrying) [inline visiting.md]:
```suggestion
```
Moved to line 33

File: visiting.md [hunk 10]
```diff
@@ -1,20 +1,60 @@
...
+1. Avoid these hosts identified by a local nonprofit as AirBnB "ghost hotel" landlords.
+    
+Please feel free to use this text for your conference in Toronto.
+
+---
+
+Would you like to make a guide like this for your event or city? If so:
+
+- Search for reports from local housing justice organizations. Read them carefully, summarize their insights, and link to them. Use their recommendations to craft your suggestions.
+- Categories of hotels to look for include: unionized hotels, small businesses, indigenous-owned hotels, black-owned hotels, as well as latino- and POC-owned hotels. There are U.S lists (see [4S 2019 in New Orleans](https://www.4s2019.org/accommodation/) conference for more), however for Canada we did a lot of searching based on cit
```
Comment (garrying) [inline visiting.md]: Not sure about the end of this sentence: `searching based on cit` -> `searching based on city`?

File: sessions-form.html [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+      <input type="checkbox" class="mr-05" id="recording2" name="recording" value="record"><label class="fw-bold" for="recording2">The presentation can be recorded and posted on the Our Networks website under a Creative Commons License</label>
+    </div>
+    <small class="block mt-1">You have the right to withdraw consent for posting after the video is recorded. You can see an example of the previous recorded talks at: <a href="https://ournetworks.ca/recorded-talks/">https://ournetworks.ca/recorded-talks/</a></small>
+  </div>
+  <div class="mb-2">
+    <h3>Presenter Details</h3>
+    <p>Please confirm you preferred pronunciation and pronouns as presenter(s):</p>
+    <div class="flex">
+      <div class="w-33">
+        <label class="fw-bold" for="presenters">Presenter:</label>
```
Comment (garrying) [inline sessions-form.html:51]:
```suggestion
        <label class="fw-bold" for="presenters">Presenter name:</label>
```
More explicit field label

File: visiting.md (hunk 10 above)
Comment (dcwalk) [reply visiting.md]: yes, should be!

File: visiting.md (hunk 9 above)
Comment (dcwalk) [reply visiting.md]: Thanks! fixed elsewhere

File: visiting.md (hunk 10 above)
Comment (dcwalk) [reply visiting.md]: fixed locally

File: visiting.md (hunk 8 above)
Comment (dcwalk) [reply visiting.md]: did this!

File: visiting.md (hunk 6 above)
Comment (dcwalk) [reply visiting.md]: did this locally

File: visiting.md (hunk 3 above)
Comment (dcwalk) [reply visiting.md]: I'm confused about this, I think this is fixed in my last commit

File: js/main.js (hunk 2 above)
Comment (dcwalk) [reply js/main.js:49]: thanks!

**Comment by dcwalk** [comment]
I think the reason the check is failing is the bug #113, I'm going to merge because this is v/ time sensitive (bad practices!) and we can follow up with deploy issues

**Pull request merged**